src/vns_lprc.jl

```diff
@@ -0,0 +1,227 @@
+#=
+# VNS_LPRC for the RENAULT Roadef 2005 challenge
+# inspired by work of
+# Celso C. Ribeiro, Daniel Aloise, Thiago F. Noronha, Caroline Rocha, Sebastián Urrutia
+#
+# @Author Boualem Lamraoui, Benoît Le Badezet, Benoit Loger, Jonathan Fontaine, Killian Fretaud, Rémi Garcia
+# =#
+
+include("parser.jl")
+include("solution.jl")
+include("functions.jl")
+include("constants.jl")
```

Comment:
Drop those include and add `include("vns_lprc.jl")` in `main.jl`

Reply:
Then this file couldn't run by it self, main shouldn't call files dependency for other files.

Reply:
I checked in the [documentation](https://docs.julialang.org/en/v1/manual/code-loading/) and a file is evaluated each times an `include` call it. So i will delete those include.